devices/Paexo/src/Paexo.cpp

```diff
@@ -439,9 +442,10 @@ bool Paexo::open(yarp::os::Searchable& config)
     // Initialize yarp control ports
 
     // Check yarp network initialization
-    if (!yarp::os::Network::isNetworkInitialized()) {
-        yInfo() << LogPrefix << "Initializing yarp network";
-        yarp::os::Network();
+    pImpl->network = yarp::os::Network();
```

Comment:
I don't this this is required, the `network` object is already allocated with the default constructor in the impl's class constructor.

Reply:
Another approach is using a unique pointer as done in https://github.com/robotology/wb-toolbox/commit/779c30771c926e6e2e38781fba06e16a2e4dc127

Reply:
> I don't this this is required, the network object is already allocated with the default constructor in the impl's class constructor.

Used the [reference](https://github.com/robotology/wearables/blob/c6a8478343881468b91112244bd25c6a803d456e/devices/IWearRemapper/src/IWearRemapper.cpp#L207) from `IWearRemapper` device.

Reply:
Not everything done in the past is strictly right :) Reviewing this code after a couple of years from when I developed the code you linked, I'm not sure it is necessary to re-create network.

Reply:
Honestly, I wouldn't touch the network related code now precisely because the testing of this can take a while as we are blocked by the hardware for the time being. So, if this is also not a blocking comment I can proceed with the PR 😉 Let me know.

Reply:
Not blocking, likely useless call.

Reply:
> Another approach is using a unique pointer as done in robotology/wb-toolbox@779c307

Done in https://github.com/robotology/wearables/pull/124/commits/24ed9be184781717f1d3db234e46595aca445755, and tested with hardware.